test/controllers/helpers/suggestionHelpersTest.js

```diff
@@ -53,7 +53,7 @@ describe('suggestionHelpers', () => {
     };
 
     const yearOfBirth = 1990
-    const currentAge = moment().year() - yearOfBirth
+    const currentAge = moment().year() - yearOfBirth - 1
```

Comment:
had to hack around here to get it to pass, but we're shortly going to remove this project so not worrying about it